kubernetes/outreach.libsonnet

```diff
@@ -42,6 +42,7 @@ k + kubecfg {
       'ingress.kubernetes.io/force-ssl-redirect': 'true',
       'kubernetes.io/tls-acme': 'true',
       'contour.heptio.com/tls-minimum-protocol-version': '1.2',
+      'projectcontour.io/tls-minimum-protocol-version': '1.2',
```

Comment:
nit

```suggestion
      'projectcontour.io/tls-minimum-protocol-version': self['contour.heptio.com/tls-minimum-protocol-version'],
```

Reply:
I think the redundant version is slightly easier to read personally, but I don't have a strong opinion on this either way.